### PR TITLE
add retries for flaky test

### LINKED ISSUE #346: Make playwright tests not flaky
I keep seeing broken playwright tests in PRs

I don't know why it's flaky, but we shouldn't be seeing that so often in PRs

/bounty $5

### PR DESCRIPTION
**Pull request by techmannih**
close #346

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { defineConfig } from "@playwright/test"
 
 export default defineConfig({
   // Run your local dev server before starting the tests
+  retries: 5,
```
Comment (seveibar) [inline playwright.config.ts:5]: Can you only configure retries on ci

Comment (seveibar) [reply playwright.config.ts:5]: Use process.env.CI ? 5 : 1

**Comment by Anshgrover23** [comment]
@seveibar i think using retries can increased overall test suite time: If tests are retried frequently.

**Comment by Anshgrover23** [comment]
image

## Also You can see it is retrying one more time even after tests are pass.

**Review by seveibar — APPROVED** [review]
@Anshgrover23 nice investigation but 2.2m vs 2.1m seems like noise not a notable difference, i could be wrong but increasing timeout rarely improves test times

**Pull request merged**